### PR TITLE
Implement ceph-ansible pipeline

### PR DESCRIPTION
**Pull request by djgalloway**

### CODE REVIEW

File: ceph-ansible-docs-prs/config/definitions/ceph-ansible-docs-prs.yml [hunk 1]
```diff
@@ -28,7 +28,8 @@
           org-list:
             - ceph
           trigger-phrase: 'jenkins test docs'
-          only-trigger-phrase: false
+          # This is set so the job can be manually triggered or by the ceph-ansible-pipeline multijob
+          only-trigger-phrase: true
```
Comment (alfredodeza) [inline ceph-ansible-docs-prs/config/definitions/ceph-ansible-docs-prs.yml]: this doesn't read right (probably my misunderstanding), but I thought `only-trigger-phrase: false` meant you can trigger with a comment in github *as well as other triggers*. Are you sure this is doing what the comment suggests?

Comment (djgalloway) [reply ceph-ansible-docs-prs/config/definitions/ceph-ansible-docs-prs.yml]: `only-trigger-phrase: false` means the job runs for every PR that gets opened/repushed was my understanding.

Comment (djgalloway) [reply ceph-ansible-docs-prs/config/definitions/ceph-ansible-docs-prs.yml]: Now I'm thinking I probably need `permit-all` set to false as well.

Comment (djgalloway) [reply ceph-ansible-docs-prs/config/definitions/ceph-ansible-docs-prs.yml]: `permit-all` just allows any Github user to trigger the job.  It should be false anyway.

But to further clarify my response, `only-trigger-phrase: true` means only the trigger phrase can trigger the job **as far as the Github Pull Request Builder plugin is concerned**.  The job can still be triggered manually by another job.

**Comment by djgalloway** [comment]
@leseb Sure, I can merge Sunday.

**Pull request merged**